main.py

```diff
@@ -23,71 +36,67 @@ def get_optimizer(model, optimizer_type, lr, momentum, weight_decay):
     if optimizer_type == "SGD":
         return optim.SGD(model.parameters(), lr=lr,
                       momentum=momentum, weight_decay=weight_decay)
-    
     elif optimizer_type == "Adam":
         return optim.Adam(model.parameters(), lr=lr, weight_decay=weight_decay)
-    
     elif optimizer_type =="adadelta":
         return optim.Adadelta(model.parameters(), lr=lr, weight_decay=weight_decay)
 
-    
-
 def get_scheduler(optimizer, scheduler_type):
     if scheduler_type == "ExponentialLR":
         return optim.lr_scheduler.ExponentialLR(optimizer, gamma=0.9)
     elif scheduler_type == "MultiStepLR":
         return optim.lr_scheduler.MultiStepLR(optimizer, milestones=[5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60], gamma=0.9)
-    
+    elif scheduler_type == "CosineAnnealingLR":
```

Comment:
mind just adding this to the readme under scheduler options?

Reply:
Sure